File: 프로그래머스/lv0/120844. 배열 회전시키기/배열 회전시키기.py
def solution(numbers, direction):
    if direction=="right":
        #저장
        num = numbers[len(numbers)-1]
        print(num)
        #당겨
        for i in range(len(numbers)-1, 0, -1):
            numbers[i] = numbers[i-1]
        #삽입
        numbers[0] = num
    else:
        num = numbers[0]
        for i in range(0, len(numbers)-1):
            numbers[i] = numbers[i+1]
        numbers[len(numbers)-1] = num   
    return numbers

File: 프로그래머스/lv0/120844. 배열 회전시키기/배열 회전시키기.js
function right(numbers){
    var answer = [];
    answer = numbers.slice(0,numbers.length-1);
    answer.unshift(numbers[numbers.length-1]);
    return answer;
}

function left(numbers){
    var answer = [];
    answer = numbers.slice(1,numbers.length);
    answer.push(numbers[0]);
    return answer;
}
function solution(numbers, direction) {
    var answer = [];
    answer = (direction==="right")? right(numbers): left(numbers);
    return answer;
}